### ISSUE TITLE
Font is really small 

### ISSUE THREAD

**Issue by KittyChoo** [MesquiteProject/MesquiteCore#69]
Tried both Java 32 and 64 bit, and the older version of Mesquite

![capture](https://cloud.githubusercontent.com/assets/28733074/26110094/d9a1b2ee-3a83-11e7-8ba5-6201e628e5cc.PNG)

**Comment by balasink** [comment]
I have the same problem - did you find a fix?

**Comment by KittyChoo** [comment]
Didn't manage to. Used it on Mac instead.

**Comment by DMaddison** [comment]
With that window at the front, you can choose Edit>Font Size to adjust the font size in this window.  If you are concerned about the small size of the menus, that is a bug in Java on Windows, and it is very difficult for us to build a workaround for it.

**Comment by KittyChoo** [comment]
Thanks for your reply. I did try to increase the font size, but the menu buttons were still very small. Thanks for letting us know it's a bug, at least I know I'll be wasting time trying to mess around to fix it.

**Comment by balasink** [comment]
Still broken.

**Comment by Injuis** [comment]
Yep.  If you browse the code, the menu font size is hard coded to 10 pt!  Tiny on anything bigger than 1080p.
They could either program a font size changer or implement a dynamic font size based on screen resolution.  Something like
`Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
double screenWidth = screenSize.getWidth();
setFont(new Font ("SanSerif", Font.PLAIN, Math.max(10, Math.round(screenWidth/100)));`

**Comment by DMaddison** [comment]
@Injuis The menu font size is not hard coded.  In fact, I don't think we touch menu font size at all.  The font size of many _windows_ (including the one shown in the initial issue post) begins at 10 point, but it user-adjustable.   When I last looked into this, this issue of small menus was caused by a bug in Java in Windows, and, at the time, there was no workaround I could find for this.

**Comment by Injuis** [comment]
Apologies, the line I looked at turned out to have done nothing after compiling the source myself.  I was, however, able to change the fonts sizes of the tabs, and submenus, just not the File, Edit, View, and Help menubar items.  From what I've read, you'd have to switch to Swing to get that to work.

**Comment by DMaddison** [comment]
@Injuis Yes, Swing would do it.  We probably need to make that change, as Oracle is not putting resources into fixing the many AWT bugs.  However, that would require a serious chunk of dedicated time, as there is so much AWT code in Mesquite (as we started it well before Swing).  I'm hoping to work on that switch this fall, but we shall see...

**Comment by AEgit** [comment]
Note that Swing is deprecated and is supposed to be replaced by JavaFX.
